ci/cron/src/Main.hs

```diff
@@ -268,11 +269,19 @@ docs = do
             update_s3 temp_dir gh_versions
         reset_cloudfront
 
+bash :: Maybe FilePath -> [String] -> IO String
+bash bash_lib cmd = do
+    shell $ unlines $ concat [
+        ["bash -c '",
+         "set -euo pipefail",
+         "eval \"$(dev-env/bin/dade assist)\"",
+         Data.Maybe.maybe "" (\l -> "source \"" <> l <> "\"") bash_lib],
```

Comment:
Which parts of the bash_lib do we actually need?

Afaict, we use

1. `gcs`, this is used here and in other scripts. However, the definition is pretty short and we could just setup the key once at the beginning of this script instead of for every call and then call `gsutil` correctly.
2. `gpg_verify`, afaict this is only used in the Haskell code so I don’t really see a reason to keep this in bash.

If those are the only two keys, I think it’s worth considering getting rid of the bash stuff completely in the medium term (not in this PR). The bash/haskell hybrid fells a bit weird.